entries/r/rakuten.com.json

```
@@ -2,6 +2,9 @@
   "Rakuten": {
     "domain": "rakuten.com",
     "url": "https://www.rakuten.com/",
+    "additional-domains": [
+      "www.rakuten.co.jp"
```

As the entry should cover all subdomains it's better to not specify a subdomain.
```suggestion
      "rakuten.co.jp"
```